tests/SerializerFactory.tests.js

```diff
@@ -46,7 +46,7 @@ QUnit.test( 'registerSerializer(), newSerializerFor()', function( assert ) {
 	);
 
 	assert.ok(
-		serializerFactory.newSerializerFor( new( testSets[0].Constructor ) ).constructor
+		serializerFactory.newSerializerFor( new ( testSets[0].Constructor ) ).constructor
```

Comment:
The brackets don't do anything here and should be removed, in my opinion.

Also I believe the syntax should be `new testSets[0].Constructor()`.